mappings/net/minecraft/advancement/criterion/LocationCriterion.mapping

```diff
@@ -0,0 +1,19 @@
+CLASS net/minecraft/unmapped/C_maikngon net/minecraft/advancement/criterion/LocationCriterion
+	FIELD f_qjamvpft id Lnet/minecraft/unmapped/C_ncpywfca;
+	METHOD <init> (Lnet/minecraft/unmapped/C_ncpywfca;)V
+		ARG 1 ud
+	METHOD m_jeouaxek (Lnet/minecraft/unmapped/C_maikngon$C_zsxskcab;)Z
+		ARG 0 conditions
+	METHOD m_qjisaduw trigger (Lnet/minecraft/unmapped/C_mxrobsgg;)V
+		ARG 1 player
+	CLASS C_zsxskcab Conditions
+		METHOD m_enfvbgsy createLocation (Lnet/minecraft/unmapped/C_ypvebgnl;)Lnet/minecraft/unmapped/C_maikngon$C_zsxskcab;
+			ARG 0 location
+		METHOD m_eyeihplq createWearingItemOnFeetStandingOnBlock (Lnet/minecraft/unmapped/C_mmxmpdoq;Lnet/minecraft/unmapped/C_vorddnax;)Lnet/minecraft/unmapped/C_maikngon$C_zsxskcab;
```

Comment:
`createWearingBootsStandingOnBlock`? That's a bad name too but this feels too long and unwieldy

Reply:
It's for checking the player has a specific item on their feet while also being on a block, probably used to the sneak 100 advancement. Tbh people shouldn't really call these but I couldn't think of a better name

Reply:
```suggestion
		METHOD m_eyeihplq checkFootwearAndBlock (Lnet/minecraft/unmapped/C_mmxmpdoq;Lnet/minecraft/unmapped/C_vorddnax;)Lnet/minecraft/unmapped/C_maikngon$C_zsxskcab;
``` perhaps?